tests/object.test.ts

```diff
@@ -106,4 +106,49 @@ describe('Object', () => {
       expect(fn(null)).toEqual('no');
     });
   });
+  
+  describe('P.object.exact({...})', () => {
+    it('should only catch exact match.', () => {
+      const fn = (input: object) =>
+        match(input)
+          .with(P.object.exact({ a: P.any }), (obj) => {
+            type t = Expect<Equal<typeof obj, { a: unknown }>>;
+            return 'yes';
+          })
+          .otherwise(() => 'no');
+      
+      expect(fn({a: []})).toEqual('yes');
+      expect(fn({a: null})).toEqual('yes');
+      expect(fn({a: undefined})).toEqual('yes');
+      expect(fn({a: 5})).toEqual('yes');
```

Comment:
Makes sure to run prettier on all these files before we merge

Reply:
oh, I usually apply format on save, but the formatting tool was set on biome(which is not included in this project). changed to prettier, and applied!